app/assets/javascripts/guiders.js

```diff
@@ -0,0 +1,42 @@
+$(document).on('turbolinks:load', function() {
+  $('.js-go-link').on('click', function() {
```

Comment:
If you used `$(document).on` you could move everything in this file up out of the `turbolinks:load`.